Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
> […] and the various custom things that are needed […]

What does this refer to? If it's about all the `inreplace` and related stuff, we could as well ignore the `install` target in the `Makefile` and simply copy the two files that would be installed manually. The entire `install` method could be distilled down to:

```ruby
def install
  rm "Makefile"
  system "make", "CFLAGS=-I/usr/include/malloc", "symlinks"
  bin.install "symlinks"
  man8.install "symlinks.8"
end
```

(We do similar things for other equally simple yet useful programs.)